packages/core/_src/stm/TQueue/operations/isShutdown.ts

```diff
@@ -0,0 +1,15 @@
+import { concreteTQueue } from "@effect/core/stm/TQueue/operations/_internal/InternalTQueue"
+
+/**
+ * Checks if the queue is at capacity.
+ *
+ * @tsplus getter ets/TQueue isShutdown
+ */
+export function isShutdown<A>(self: TQueue<A>): USTM<boolean> {
+  concreteTQueue(self)
+  return STM.Effect((journal) => {
+    const queue = self.ref.unsafeGet(journal)
+
+    return queue === null
```

Comment:
Can you guarantee that this will be strictly equal to `null` if not present? 

I find that it's often better to use `x == null` for these types of checks because then you check for both `null` and `undefined`.